File: Algorithms/1. Two Sum/twoSum.c
/**
 * Note: The returned array must be malloced, assume caller calls free().
 */
int* twoSum(int* nums, int numsSize, int target) {
    int i, j;
    int *ans;
    
    ans = (int *)malloc(2*sizeof(int));
    
    for (i = 0; i < numsSize; i++) {
        for (j = 0; j < numsSize; j++) {
            if (j == i) 
                continue;
            if ((nums[i] + nums[j]) == target) {
                ans[0] = i;
                ans[1] = j;
                return ans;
            }
        }
    }
    return NULL;
}

File: Algorithms/1. Two Sum/twoSum.java
public class Solution {
    public int[] twoSum(int[] nums, int target) {
        int[] ans = new int[2];
        int i, j;
        for (i = 0; i < nums.length; i++) {
            for (j = 0; j < nums.length; j++) {
                if (i == j) continue;
                if (target == nums[i] + nums[j]) {
                    ans[0] = i;
                    ans[1] = j;
                    return ans;
                }
            }
        }
        return null;
    }
}